pkg/rc/replication_controller.go

```diff
@@ -0,0 +1,21 @@
+package rc
+
+import (
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/pods"
+)
+
+type ReplicationController interface {
+	ID() string
+	Manifest() *pods.Manifest
+	NodeSelector() labels.Selector
+	Disabled() bool
+	DesiredReplicas() int
+
```

Comment:
what else does the orchestrator need?

* `PodLabels()`? I say no.
* `CurrentNodes()`? I say yes.
* `EligibleNodes()`? I say no.